requirements-dev.txt

```diff
@@ -8,3 +8,4 @@ mock==2.0.0
 flake8==2.5.4
 flake8-import-order==0.8
 selenium==3.6.0
+groupy
```

Comment:
shouldn't we pin the version? and sort alphabetically?

Reply:
Yeah, we should sort alphabetically. As for the version, I intentionally left that unpinned so tests always run against the latest stable release.